mgmtsystem_nonconformity_mrp/models/mgmtsystem_nonconformity_mrp.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2019 Marcelo Frare (Ass. PNLUG - Gruppo Odoo <http://odoo.pnlug.it>)
+# Copyright 2019 Stefano Consolaro (Ass. PNLUG - Gruppo Odoo <http://odoo.pnlug.it>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class MgmtsystemMgmMRP(models.Model):
+    """
+    Extend nonconformity adding fields for workcenter
+    """
+
+    _inherit = ['mgmtsystem.nonconformity']
+
+    # new fields
+    # workcenter reference
```

Comment:
```suggestion
```

Reply:
I know, I tend to over comment the code, but I had a bad experience with Odoo when I begin to learn it. To help who is going inside for the first time I prefer to let this over comment.
But if there is some "special directive" to cut code at minimum, I will cut it.